.gitignore

```diff
@@ -58,7 +58,13 @@ website/static/robots.local.txt
 # OSF-specific
 ##############
 
-api/static/
+api/static/rest_framework
```

Comment:
This is related to the CSS files checked in below, which @samanehsan will probably explain better than I.